### DISCUSSION TITLE
xquic的test_client和test_server的测试结果

### DISCUSSION THREAD

**Discussion by david-hshu** [alibaba/xquic#120] [Q&A] [answered]
speed的单位是什么 Kbit/s吗？
如果是Kbit/s我的测试结果带宽利用率非常低 不会超过20%，网络环境波动后就会更低
请问xquic的实际测试速度是多少呢

**Comment by adcen0107** [comment, marked as answer]
看代码speed的单位应该是 Kbyte/s，我在千兆局域网测的单流bbr的吞吐达到了200Mbps。

**Reply by ginger0106** [reply]
> 看代码speed的单位应该是 Kbyte/s，我在千兆局域网测的单流bbr的吞吐达到了200Mbps。

我只看到代码写了K/s， 请问是怎么确定的是byte呢？

**Reply by adcen0107** [reply]
> 
它的计算过程是 字节数*1000/us

**Reply by wangweiwei1188** [reply]
> 看代码speed的单位应该是 Kbyte/s，我在千兆局域网测的单流bbr的吞吐达到了200Mbps。
在千兆局域网测的单流bbr的吞吐达到了200Mbps。
---------------------------------------------------------------------------------------
可以把测试代码贴出来嘛？性能只有200Mbps，性能是不是有点低？

**Comment by ghost** [comment]
我测试也是最大200M/bps
